### ISSUE TITLE
HUD error

### ISSUE THREAD

**Issue by ViralBrain** [bmino/binance-triangle-arbitrage#30]
**When I enter npm start, I see this error:
(my npm is 6.7.0 and node is 11.8.0)**

```
> binance-triangle-arbitrage@4.4.1 start /Users/tom_ryzhkov/Desktop/binance-triangle-arbitrage-master
> node src/main/Main.js

Found 440/468 currently trading tickers.
Checking configuration ...
Opening 440 depth websockets ...

Execution Strategy:     linear
Optimization Ticks:     4
Execution Limit:        1 execution(s)
Profit Threshold:       0.30%
Age Threshold:          300 ms
Log Level:              info

TypeError: Cannot read property 'a' of undefined
    at arbs.forEach.arb (/Users/tom_ryzhkov/Desktop/binance-triangle-arbitrage-master/src/main/HUD.js:50:31)
    at Array.forEach (<anonymous>)
    at Object.displayArbs (/Users/tom_ryzhkov/Desktop/binance-triangle-arbitrage-master/src/main/HUD.js:48:14)
    at refreshHUD (/Users/tom_ryzhkov/Desktop/binance-triangle-arbitrage-master/src/main/Main.js:137:9)
    at Timeout.calculateArbitrage [as _onTimeout] (/Users/tom_ryzhkov/Desktop/binance-triangle-arbitrage-master/src/main/Main.js:78:29)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! binance-triangle-arbitrage@4.4.1 start: `node src/main/Main.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the binance-triangle-arbitrage@4.4.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tom_ryzhkov/.npm/_logs/2019-02-17T12_27_24_016Z-debug.log
```

**What could be the problem?
Thanks.**

**Comment by bmino** [comment]
Sure looks like a code error. I'm looking into it now

**Comment by bmino** [comment]
Fixed in 1941dd0dc1be9d1b7a6ad14d66046f6d7cf19626
Will be delivered in v4.4.2 shortly

**Comment by bmino** [comment]
Thanks for the heads up!

Please confirm this is fixed for you in release v4.4.2

**Comment by ViralBrain** [comment]
Yes, it works!
Thanks.
